` !  ~/D/g/nodejs-sdk  feat/update_node_14  >  npm run manualtest                                                                                                                                         

> @devo/nodejs-sdk@2.3.4 manualtest
> mocha --reporter list --timeout 50000 test/manual/*.js --exit


  ✔ Query client queries remote server: 814ms
  ✔ Query client queries with overriden env variable: 110ms
  ✔ Query client queries with url ending in /: 703ms
  ✔ Query client queries for CSV: 717ms
  ✔ Query client queries for full JSON: 613ms
  ✔ Query client queries with skip and limit: 664ms
  ✔ Query client streams a query: 647ms
  1) Query client streams without finish date
  ✔ Query client streams with invalid query: 466ms
  ✔ Query client streams with invalid table: 534ms
  2) Tasks creates a new task and deletes it
  3) Tasks starts and stops a new task
  ✔ Tasks gets the list of tasks: 362ms

  10 passing (12s)
  3 failing

  1) Query client
       streams without finish date:
     Uncaught AssertionError: expected 0 to be above 0
      at Assertion.fail (node_modules/should/cjs/should.js:275:17)
      at Assertion.value (node_modules/should/cjs/should.js:356:19)
      at Timeout._onTimeout (test/manual/query.js:140:29)
      at listOnTimeout (internal/timers.js:557:17)
      at processTimers (internal/timers.js:500:7)

  2) Tasks
       creates a new task and deletes it:
     RequestError: Invalid status code 400
      at /home/joserafonso/Devo/github/nodejs-sdk/node_modules/basic-request/lib/request.js:50:30
      at /home/joserafonso/Devo/github/nodejs-sdk/node_modules/basic-request/lib/request.js:76:10
      at ClientRequest.<anonymous> (node_modules/basic-request/lib/parse.js:64:12)
      at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:647:27)
      at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
      at TLSSocket.socketOnData (_http_client.js:515:22)
      at addChunk (internal/streams/readable.js:293:12)
      at readableAddChunk (internal/streams/readable.js:267:9)
      at TLSSocket.Readable.push (internal/streams/readable.js:206:10)
      at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)

  3) Tasks
       starts and stops a new task:
     RequestError: Invalid status code 400
      at /home/joserafonso/Devo/github/nodejs-sdk/node_modules/basic-request/lib/request.js:50:30
      at /home/joserafonso/Devo/github/nodejs-sdk/node_modules/basic-request/lib/request.js:76:10
      at ClientRequest.<anonymous> (node_modules/basic-request/lib/parse.js:64:12)
      at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:647:27)
      at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
      at TLSSocket.socketOnData (_http_client.js:515:22)
      at addChunk (internal/streams/readable.js:293:12)
      at readableAddChunk (internal/streams/readable.js:267:9)
      at TLSSocket.Readable.push (internal/streams/readable.js:206:10)
      at TLSWrap.onStreamRead (internal/stream_base_commons.js:188:23)`

--- a/README.md
+++ b/README.md
@@ -90,6 +90,22 @@ Note: you will need to have your
 in the file
 `$HOME/.devo.json`.
 
+In the task test is necessary to change the mail.
+``` js
+const options = {
+  dateFrom: new Date(Date.now() - 60 * 1000),
+  dateTo: new Date(),
+  query: QUERY,
+  destination: {
+    type: 'email',
+    params: {
+      'email.to': 'my@mail.com',
+      'email.subject': 'test'
+    }
+  },
+}
+```
+
 And start playing!
 Pull requests are welcome ☺
 

